python/src/aiconfig/editor/client/src/utils/constants.ts

```diff
@@ -1 +1 @@
-export const DEBOUNCE_MS = 250;
+export const DEBOUNCE_MS = 300;
```

Comment:
Why change this?

Reply:
I was playing around with it when testing the bug with prompt name that you fixed so this is technically not needed. However, I'd like to keep this because it doesn't result in any noticeable UI change while also potentially reducing the amount of requests sent. With debounce you sometimes need to play around with it a bit to find the sweet spot